Marlin/src/gcode/feature/pause/M600.cpp

```diff
@@ -71,42 +101,101 @@ void GcodeSuite::M600() {
   const float x_pos = parser.linearval('X', 0
     #ifdef PAUSE_PARK_X_POS
       + PAUSE_PARK_X_POS
+	#else
+      + current_position[X_AXIS]
     #endif
```

Comment:
MY FAVORITE   
If XPOS or YPOS not defined , the pause moves only one axis
For example ' a printer with movng bed ' and purge bucket on the right , only the x carriage need to move , and bed can stay off